yogstation/code/game/objects/items/shields.dm

```diff
@@ -0,0 +1,8 @@
+/obj/item/shield/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
+	SEND_SIGNAL(src, COMSIG_ITEM_HIT_REACT, args)
+	if(transparent && (hitby.pass_flags & PASSGLASS))
+		return FALSE
+	if(!is_A_facing_B(owner, hitby))
+		return FALSE
+	owner.visible_message("<span class='danger'>[owner] blocks [attack_text] with [src]!</span>")
+	return TRUE
```

Comment:
Why did you have to mirror this proc with no call to ``..()``?

Reply:
Because otherwise it would block things that aren't in front, because of the default behaviour of the hit_reaction proc. I put everything that otherwise gets handled by the proc in this one, so it's not necessary.